Java/AutoML/automl-cms-api/src/main/java/com/automl/cms/api/controller/MLDatasetController.java

```diff
@@ -0,0 +1,50 @@
+package com.automl.cms.api.controller;
```

Comment:
Will the code become clearer to remove all "ML" prefixes?

Reply:
Yes, com.ml. Automl is a bit long

I do it based on the name of the project

Reply:
um. I mean just Dataset will be ok, the ML prefix could be omitted.

Reply:
According to my discovery on other projects, they don't need a special prefix when doing the module design or database design. Why will it be confusing without 'ML' prefix?
![image](https://user-images.githubusercontent.com/98159352/185524500-792ade7e-989a-4b1c-84c6-9693f61be9c8.png)

Reply:
Maybe it's a matter of personal thinking

The Run class alone has no meaning and no business context.
Ml can feel like this class is a machine learning model, and Job is a machine learning model.
You can specify the writing rules, if you remove the ML prefix, I will propose a new submission to you

Reply:
I think that it's a machine learning project, we don't need a prefix 'ML' to understand that those resources are machine learning related concepts.

Reply:
Will the table names in the database be changed as well?

It always feels like there is a lack of business meaning, making entities meaningless.

or, you can create  a set of rules.

Reply:
For the table name, we can use Dataset and other entity names directly. Or we can keep the 'ML' prefix, it depends on your design.

As regards the meaning, this is a machine learning project, we all know the entities' meaning and context. We know the meaning not only from the project, but also know it as a machine learning database from the database name.

Reply:
Okay, so let's just keep the ML prefix

Reply:
Great! Except for database tables, please omit all ML prefixes so that our code and module naming could be clearer.